### DISCUSSION TITLE
Ultra Street Fighter 2

### DISCUSSION THREAD

**Discussion by MrSujano** [skyline-emu/skyline#501] [Game Requests]
Ultra Street Fighter 2

**Comment by Pervasive** [comment]
Lmao

**Reply by MK11Shadow** [reply]
Mr Sujano is legit 😂💯

**Reply by enthiran01** [reply]
Yes sir, of course this should run well on mobile devices.

**Reply by VoidOfficial421** [reply]
Subscribe to Mr. Sujano

**Reply by VAILL666** [reply]
skyline mmj

**Reply by enthiran01** [reply]
If this game runs, I will love it and will no longer need it on dreamcast as it has commandlist and moves in training mode.

**Comment by limwer** [comment]
my favorite YouTuber is so cool can you post in the morning pls keep up good work and subscribe to mr sujano like me
